pubspec.yaml

```diff
@@ -12,7 +12,7 @@ homepage: https://github.com/Workiva/dart_dev
 dependencies:
   ansicolor: "^0.0.9"
   args: "^0.13.0"
-  coverage: "^0.7.2"
+  coverage: "^0.7.3"
```

Comment:
we should update the readme to document the change in the recommended dependencies https://github.com/Workiva/dart_dev#install-dart_dev